src/Cli/func/Actions/AzureActions/LogStreamAction.cs

```diff
@@ -44,9 +45,9 @@ public override async Task RunAsync()
                 return;
             }
 
-            if (functionApp.IsLinux && functionApp.IsDynamic)
+            if (functionApp.IsFlex || (functionApp.IsLinux && functionApp.IsDynamic))
             {
-                throw new CliException("Log stream is not currently supported in Linux Consumption Apps. " +
+                throw new CliException("Log stream is not currently supported in Linux Consumption and Flex Apps. " +
```

Comment:
Don't we need any update on the tests covering this public method for the change this exception message?

Reply:
No, there's no test around this.